doc/getting-started/rt_industry_ubuntu.rst

```diff
@@ -351,19 +353,18 @@ Follow these instructions to build the RT kernel.
 
    .. code-block:: none
 
+      $ cp x86-64_defconfig .config
       $ make olddefconfig
       $ make targz-pkg
 
 #. Copy the kernel and modules:
 
    .. code-block:: none
 
-      $ sudo mount /dev/sda1 /mnt
-      $ sudo cp arch/x86/boot/bzImage /mnt/EFI/
-      $ sudo umount /mnt
       $ sudo mount /dev/sda2 /mnt
-      $ sudo cp linux-4.19.72-rt25-x86.tar.gz -P /mnt/usr/lib/modules/ && cd /mnt/usr/lib/modules/
-      $ sudo tar zxvf linux-4.19.72-rt25-x86.tar.gz
+      $ sudo cp arch/x86/boot/bzImage /mnt/boot/
```

Comment:
This seems very suspicious... is the content of the `/boot` folder really on `/dev/sda2`??

Reply:
sda     8:0    0 119.2G  0 disk
├─sda1 8:1    0   762M  0 part /boot/efi
├─sda2 8:2    0 111.8G  0 part /
└─sda3 8:3    0   6.8G  0 part [SWAP]
only /boot/efi is in /dev/sda1, others of /boot all in /dev/sda2.

Reply:
Ah yes, I installed Ubuntu over the week-end and now I see this. Thanks!